dev/linters/helmlint.sh

```diff
@@ -2,7 +2,7 @@
 
 set -o errexit -o nounset
 
-if grep --exclude="helmlint.sh" -r "{{-.*-}}" .; then
+if git grep --quiet '{{-.*-}}' -- . ':!/deploy/helm/kubecf/charts' ':!/dev/linters/helmlint.sh'; then
```

Comment:
Could we either user `git -c core.pager=cat` or `PAGER=cat` or something instead? Not being able to see _where_ the issue is will be very annoying.